pkgs/applications/audio/rkrlv2/default.nix

```diff
@@ -1,26 +1,25 @@
-{ stdenv, fetchFromGitHub,
-automake, pkgconfig, lv2, fftw, cmake, xorg, libjack2, libsamplerate, libsndfile
-}:
+{ stdenv, fetchFromGitHub, pkg-config, lv2, fftw, cmake, libXpm
+, libXft, libjack2, libsamplerate, libsndfile }:
 
 stdenv.mkDerivation rec {
-  repo = "rkrlv2";
-  name = "${repo}-b2.0";
+  pname = "rkrlv2";
+  version = "beta_3";
 
   src = fetchFromGitHub {
     owner = "ssj71";
-    inherit repo;
```

Comment:
That version bump needs to go into a separate commit.

Reply:
Thanks.